fluxfw-gcp/tf/examples/foundation/Makefile

```diff
@@ -0,0 +1,43 @@
+NETWORK_STATE = network/terraform.tfstate
+IAM_STATE     = iam/terraform.tfstate
+STORAGE_STATE = storage/terraform.tfstate
```

Comment:
is there a way to make this makefile useful to a user who wants to "adopt" (copy/paste) this?  remote state's gonna be one of the first things they'll do on thier own project.  No-op, just thinking out-loud.